osg_configure/configure_modules/gratia.py

```diff
@@ -618,89 +426,72 @@ def _configure_htcondor_ce_probe(self):
-            data_folder = match.group(1)
-            data_folder = data_folder.strip('" \t')
-            # PER_JOB_HISTORY_DIR comes from the schedd, so if condor's not
-            # running, we can't get a value (SOFTWARE-1564)
-            history_dir = self._get_condor_history_dir(condor_config_val_bin)
-            if not history_dir:
-                self.log("Could not verify DataFolder correctness: unable to get PER_JOB_HISTORY_DIR. "
-                         "This may be caused by the condor schedd not running, or by PER_JOB_HISTORY_DIR "
-                         "not being defined.", level=logging.WARNING)
-            else:
-                # os.path.samefile will die if the paths don't exist so check that explicitly (SOFTWARE-1735)
-                if not os.path.exists(data_folder):
-                    self.log("DataFolder setting in %s (%s) points to a nonexistant location" % (
-                    config_location, data_folder),
-                             level=logging.ERROR)
-                    valid = False
-                elif not os.path.exists(history_dir):
-                    self.log("Condor PER_JOB_HISTORY_DIR %s points to a nonexistant location" % history_dir,
-                             level=logging.ERROR)
-                    valid = False
-                else:
-                    try:
-                        if not os.path.samefile(data_folder, history_dir):
-                            self.log("DataFolder setting in %s (%s) and condor PER_JOB_HISTORY_DIR %s "
-                                     "do not match, these settings must match!" % (config_location,
-                                                                                   data_folder,
-                                                                                   history_dir),
-                                     level=logging.ERROR)
-                            valid = False
-                    except OSError as e:
-                        self.log("Error comparing DataFolder setting in %s (%s) and condor PER_JOB_HISTORY_DIR %s:\n%s"
-                                 % (config_location, data_folder, history_dir, e),
-                                 level=logging.ERROR)
-                        valid = False
-
-            # Per Gratia-126 DataFolder must end in / otherwise gratia won't find certinfo files
-            if not data_folder.endswith('/'):
-                self.log("DataFolder setting in %s must end in a /" % config_location,
-                         level=logging.ERROR)
-                valid = False
+        if not match:
+            return True
 
-        return valid
+        data_folder = match.group(1)
+        data_folder = data_folder.strip('" \t')
+        # Per Gratia-126 DataFolder must end in / otherwise gratia won't find certinfo files
+        if not data_folder.endswith('/'):
+            self.logger.error("DataFolder setting in %s must end in a /", config_location)
+            return False
```

Comment:
Ugh is this true? We may have to cut a new `gratia-probe` build: https://github.com/opensciencegrid/gratia-probe/pull/93/files#diff-0315e0f76461c232d607a9abc31059f537460469f5e4adbdd2c33efdf8c47b29R289

Reply:
I don't think this is necessarily true but it looks like we have to make cut a new gratia-probe for other reasons anyway so I can ensure that the dirs have the trailing slashes

Reply:
It's not true. `condor_meter` in gratia-probes performs the same check: https://github.com/opensciencegrid/gratia-probe/blob/master/condor/condor_meter#L877-L880 but it's bogus there too -- gratia-probes very sensibly uses `os.path.join()` wherever DataFolder is used.

Reply:
Ah and the other dirs that we're setting in the ProbeConfig also construct paths with `os.path.join()`. Phew